README.md

```diff
@@ -149,7 +149,7 @@ You'll need Ruby v2.4.6 (use your local ruby version management system - RVM / r
 5. Initialize database with `bundle exec rails db:setup`
 6. Enter ReCaptcha public and private keys in config/initializers/recaptcha.rb, copied from recaptcha.rb.example. To get keys, visit https://www.google.com/recaptcha/admin/create
 7. Install static assets (like external javascript libraries, fonts) with `yarn install` 
-8. Start rails with `bundle exec passenger start` from the Rails root and open http://localhost:3000 in a web browser. (For some, just `passenger start` will work; adding `bundle exec` ensures you're using the version of passenger you just installed with Bundler.)
+8. Start rails with `bundle exec rails s` from the Rails root and open http://localhost:3000 in a web browser. (For some, just `rails s` will work; adding `bundle exec` ensures you're using the version of passenger you just installed with Bundler.)
```

Comment:
we should add [forego](https://github.com/ddollar/forego#forego) as a dependency. With this we should start MK with `forego start` to run both Puma and Passenger.

Reply:
@alaxalves can we handle that in a follow up pr? is there an issue open for it? Everything works fine with it so if its not critical id rather not add it on here.

Reply:
Of course! Also it could be a FTO issue. :smile: :smile: :smile: